prusti-interface/src/environment/mod.rs

```diff
@@ -389,6 +389,7 @@ impl<'tcx> Environment<'tcx> {
     /// Checks whether the given type implements the trait with the given DefId.
     /// Accounts for generic params on the trait by the given substs.
     /// TODO hansenj: name, params, docs
+    /// TODO hansenj: Do we need this?
```

Comment:
I assume yes? A ghost constraint `T: Foo<i32>` is not the same as `T: Foo<bool>`.